LISISO/RPMS65/LIS52/install.sh

```diff
@@ -40,7 +40,7 @@ if [ "$kmodrpm" != "" ] && [ "$msrpm" != ""  ]; then
        rpm -ivh $kmodrpm
        kmodexit=$?
        if [ "$kmodexit" == 0 ]; then
-              rpm -ivh --nodeps $msrpm
+              rpm -ivh $msrpm
```

Comment:
Do we need to remove --nodeps option?

Reply:
Seems copy is made wrongly from CentOS52 to RPMS65. Please check

Reply:
Done